src/shared/components/Typography/styles.ts

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+import { StyledTypographyProps } from './types';
+import { css } from '@emotion/react';
+
+export const H1 = styled.h1<StyledTypographyProps>`
+  font-size: 40px;
+  font-weight: 600;
+  margin: 0;
+  ${({ color, textAlign }) => css`
+    color: ${color};
+    text-align: ${textAlign};
+  `}
+`;
+
+export const H2 = styled.h2<StyledTypographyProps>`
+  font-size: 32px;
+  font-weight: 600;
+  margin: 0;
+  ${({ color, textAlign }) => css`
+    color: ${color};
+    text-align: ${textAlign};
+  `}
```

Comment:
**Question:** Is it possible to extract it to a variable?

Reply:
I did it as a function, let me know if you liked it.